### PR TITLE
Create event resource

### PR DESCRIPTION
**Pull request by JopaFM**
This PR creates event resource, including model and basic endpoints.

### CODE REVIEW

File: app/controllers/v1/events_controller.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    def index
+      @events = Event.all
+
+      render json: {
+        events: @events
+      }
+    end
+
+    def show
+      @event = Event.find(params[:id])
```
Comment (arturcp) [inline app/controllers/v1/events_controller.rb:12]: We can create a hook to remove this `find` from the show, update and destroy actions. Something like this:

```
class EventsController < ApplicationController
  before_action :load_event

  ...

  private

  def load_event
    @event ||= Event.find(params[:id])
  end
end
```

File: app/models/event.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class Event < ApplicationRecord
+    self.inheritance_column = nil
```
Comment (arturcp) [inline app/models/event.rb:4]:
```suggestion
  self.inheritance_column = nil
```

File: db/seed/data/events.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+:event_1:
```
Comment (arturcp) [inline db/seed/data/events.yml:1]: I think that we don't need the `:` before each word. 

```
  event_1:
    type: "Party"
   ...
```

Comment (JopaFM) [reply db/seed/data/events.yml]: Done! We really don't need the `:`

File: app/controllers/v1/events_controller.rb (hunk 1 above)
Comment (JopaFM) [reply app/controllers/v1/events_controller.rb]: Nice tip!! Much better now :)

File: db/schema.rb [hunk 4]
```diff
@@ -45,6 +45,18 @@
...
   end
 
+  create_table "events", force: :cascade do |t|
+    t.string "type"
+    t.string "title"
+    t.string "description"
+    t.string "link"
+    t.string "date"
+    t.string "start_time"
+    t.string "end_time"
```
Comment (gabriel-murakami) [inline db/schema.rb:55]: why `date`, `start_time` and `end_time` are not a datetime instead of string type?

**Pull request merged**